.travis.yml

```diff
@@ -28,6 +28,10 @@ branches:
     - master
     - /^greenkeeper/.*$/
 
+before_script:
+    "google-chrome --product-version"
+    "echo google-chrome --product-version"
```

Comment:
are these necessary?

Reply:
sorry I was checking which version of chrome was present